requirements.txt

```diff
@@ -1,20 +1,28 @@
 astroid==2.0.1
 autopep8==1.3.5
+certifi==2018.4.16
+chardet==3.0.4
 coverage==4.5.1
+coveralls==1.3.0
 Django==2.0.6
 django-cors-middleware==1.3.1
 django-coverage==1.2.4
 django-extensions==2.1.0
 django-nose==1.4.5
 djangorestframework==3.8.2
 djangorestframework-jwt==1.11.0
+docopt==0.6.2
+idna==2.7
```

Comment:
why did you remove psycopg2-binary yet it is required

Reply:
My bad, I just did a `pip freeze > requirements.txt`. I guess it was not in the requirements but let rectify this